src/main/java/com/github/markusbernhardt/seleniumlibrary/locators/TableElementFinder.java

```diff
@@ -67,23 +64,20 @@ public static WebElement findByRow(WebDriver webDriver, String tableLocator, int
 
 	public static WebElement findByCol(WebDriver webDriver, String tableLocator, int col, String content) {
 		List<String> locators = parseTableLocator(tableLocator, "col");
-		List<String> formattedLocators = new ArrayList<String>();
+		List<String> formattedLocators = new ArrayList<>();
 		for (String locator : locators) {
 			formattedLocators.add(String.format(locator, Integer.toString(col)));
 		}
 		return searchInLocators(webDriver, formattedLocators, content);
 	}
 
 	protected static void addLocatorSuffix(Map<String, List<String>> locatorSuffixesMap, String key, String... values) {
-		List<String> list = new ArrayList<String>();
-		for (String value : values) {
-			list.add(value);
-		}
+		List<String> list = new ArrayList<>(Arrays.asList(values));
```

Comment:
Collections.addAll()?

Reply:
Done

Reply:
Could you explain to me why this is better?

Reply:
`Arrays.asList()` creates a new ArrayList under the hood. After that, we pass it to a ArrayList() constructor where we create an array (see `toArray()` there), and after that we copy this array to a destination array. So, here we've copied elements 2 times and produced unnecessary temporary objects (more work to GC). We can avoid this by using `Collections.addAll()`. That was my point.

Reply:
ok, thanks a lot!